src/types/plugin.ts

```diff
@@ -34,6 +34,7 @@ export type IamStatement = {
 
 export type WafConfig = {
   enabled?: boolean;
+  arn?: string;
```

Comment:
please add [AJV validation](https://github.com/sid88in/serverless-appsync-plugin/blob/alpha/src/validation.ts#L659-L670) for this.

Reply:
hmm, not sure how I missed those tests. Have added them now, thanks @bboure. Do you mind checking #501 too?